notebooks/official/matching_engine/sdk_matching_engine_create_stack_overflow_embeddings_palm.ipynb

```diff
@@ -0,0 +1,1483 @@
+{
```

Comment:
<p>Change to "After you've built" or change to "After you build" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Reply:
<p>Done</p>